core/src/test/scala/unit/kafka/security/authorizer/AclAuthorizerTest.scala

```diff
@@ -100,8 +106,15 @@ class AclAuthorizerTest extends ZooKeeperTestHarness {
 
   @After
   override def tearDown(): Unit = {
-    aclAuthorizer.close()
-    aclAuthorizer2.close()
+    val authorizers = Seq(aclAuthorizer, aclAuthorizer2)
+    authorizers.foreach(a => {
+      a.acls(AclBindingFilter.ANY).forEach(bd => {
+        removeAcls(aclAuthorizer, Set(bd.entry), bd.pattern())
```

Comment:
Why do we need this in tearDown?

Reply:
Otherwise "deny all" will remain in ZK during the whole test process since ZK won't be restarted or re-instantiated.

Reply:
Removed.